service/controller/aws/v7/resource/configmap/resource.go

```diff
@@ -17,19 +16,19 @@ const (
 
 // Config represents the configuration used to create a new chart config resource.
 type Config struct {
-	ConfigMap   configmap.Interface
-	Guest       guestcluster.Interface
-	K8sClient   kubernetes.Interface
-	Logger      micrologger.Logger
+	ConfigMap configmap.Interface
+	K8sClient kubernetes.Interface
+	Logger    micrologger.Logger
+
```

Comment:
The configmap resources use the configmap service to provide all functionality. So they don't need the tenantcluster service.

Fixing this to simplify the resources.